commands/daemonize/daemonize.go

```diff
@@ -0,0 +1,70 @@
+package daemonize
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/takama/daemon"
+	cmd_ep "github.com/taskcluster/taskcluster-worker/commands/extpoints"
+	"github.com/taskcluster/taskcluster-worker/runtime"
+)
+
+const (
+	// name of the service
+	name        = "taskcluster-worker"
+	description = "Taskcluster worker"
+)
+
+var dependencies = []string{}
+
+func init() {
+	cmd_ep.CommandProviders.Register(cmd{}, "daemon")
+}
+
+type cmd struct{}
+
+func (cmd) Summary() string {
+	return "Run as a daemon."
+}
+
+func usage() string {
+	return `Usage:
+  taskcluster-worker daemon <action> [--engine <engine>] [--logging-level <level>]
+
+Options:
+  -l --logging-level <level>  	Set logging at <level>.
+  -e --engine <engine>			Set the engine at <engine>.
```

Comment:
Can you specify the possible value for `<action>` too?

I think I prefer to list them explicitly in the usage - this way, each action can have its own parameter options (as different actions require different parameters). For example, like [this](https://github.com/taskcluster/generic-worker/blob/a4784866719f50ff9e47b63c0e444839f8dc4e56/main.go#L70-L81).